wordpress-mobile/WordPress-Android
# Create toolbar in the improved My Site fragment

Fixes #13344

This PR moves the existing collapsing toolbar and Me button logic to the new `ImprovedMySiteFragment`. It also introduces a `UiModel` class to `MySiteViewModel` in order to handle avatar changes.

To test:

1. Enable the Improved My Site flag and restart the app.
1. On the My Site fragment, notice the new toolbar.
1. Verify that both the title and the Me button are resized when the toolbar is collapses and expands.
1. Press the Me button and verify that the Me screen appears.
1. Change the account's avatar, go back to the My Site fragment and verify that the avatar was updated.

PR submission checklist:
